main.c

```diff
@@ -128,7 +128,7 @@ void init_CS(void) {
 
 // Serial port reset procedure, called before any I/O and called again upon I/O error
 #if !NOTECARD_USE_I2C
-void noteSerialReset() {
+bool noteSerialReset() {
```

Comment:
Looks like there's still no return statement, though, from what I can see here.

Reply:
Good catch. I'm surprised the compiler didn't complain... I'll add all warnings as errors.